public/app/core/components/AccessControl/Permissions.tsx

```diff
@@ -266,6 +272,16 @@ const setPermission = (
 ): Promise<void> =>
   getBackendSrv().post(`/api/access-control/${resource}/${resourceId}/${type}/${typeId}`, { permission });
 
+const addTeamLBACWarnings = (teams: number[], items: ResourcePermission[]) => {
+  return items.map((item) => {
+    if (item.permission !== 'Admin' && (!item.teamId || !teams.includes(item.teamId))) {
```

Comment:
so this will show regardless of permission set, as long as it is not admin, OR the team itself.

I think this makes sense, just want to confirm.

Reply:
Yes, since Edit includes query